git_machete/tests/functional/test_machete.py

```diff
@@ -1784,7 +1787,7 @@ def test_update_with_fork_point_specified(self) -> None:
         )
 
     git_api_state_for_test_retarget_pr = MockGithubAPIState(
-        [{'head': {'ref': 'feature'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'root'}, 'number': '15',
+        [{'head': {'ref': 'feature', 'repo': {'full_name': "testing/ignore", 'html_url': 'www.github.com'}}, 'user': {'login': 'github_user'}, 'base': {'ref': 'root'}, 'number': '15',
```

Comment:
Hmmm does `html_url` in actual data from the API include just the host with no URL path? 🤔

Reply:
It outputs full url here, but from testing point of view it's irrelevant. But i'll add it anyway